docs/releasing.md

```diff
@@ -0,0 +1,69 @@
+# Overview
+
+This document describes the release flow and how to create a bundle release. 
+
+## Latest tag 
+
+Every change in the master branch which passes all checks triggers creating a `latest` tag and release.
+
+## Versioned releases
+
+Every tag pushed to the repository triggers new release. If the tag name matches the version pattern `x.y.0` (revision is `0`) Travis CI build creates a release tag `release-x.y`.
```

Comment:
Every tag pushed to the repository triggers `a` new release. ``If the tag name matches the `x.y.0` version pattern, in which `0`, Travis CI build creates a release tag named `release-x.y`.``